contrib/fuzzing/fuzz-settings.sh

```diff
@@ -15,4 +15,13 @@ FUZZING_FINDINGS_DIR=~/libopenmpt-fuzzing
 # Fuzzer timeout in ms, + = don't abort on timeout
 FUZZING_TIMEOUT=5000+
 # Path to afl-fuzz binary
-AFL_DIR=afl
+AFL_PATH=afl
+
+# AFL specific envs
+AFL_TRY_AFFINITY=1
+AFL_CMPLOG_ONLY_NEW=1
+AFL_NO_WARN_INSTABILITY=1
+AFL_FAST_CAL=1
+AFL_IMPORT_FIRST=1
+AFL_DISABLE_TRIM=1
```

Comment:
Can you explain why using `AFL_DISABLE_TRIM=1` would be beneficial? It seems counterintuitive given the afl++ documentation, and also given that a lot of our formats have the possibility of having large chunks of PCM sample data at the end which should not affect the fuzzing state at all, so trimming this data would often result in smaller files and thus faster fuzzing cycles according to my understanding.

Reply:
In >60% of the time it’s better that is why :) you could try it out yourself by running it with and without for 3h